content/SCALE/SCALECLIReference/Service/_index.md

```diff
@@ -3,9 +3,355 @@ title: "Service"
+
+{{< expand "Running a Simple Query" "v" >}}
+The `query` command has no additional requirements. After entering the command, it returns a table with multiple outputs.
+
+From the CLI prompt, enter:
+
+<code>service query</code>
+
+From the **service** prompt, enter:
+
+<code>query</code>
+
+{{< expand "Command Example" "v" >}}
+```
+service> query
++----+-------------+--------+---------+--------------+
+| id | service     | enable | state   | pids         |
++----+-------------+--------+---------+--------------+
+| 4  | cifs        | false  | STOPPED | <empty list> |
+| 6  | ftp         | false  | STOPPED | <empty list> |
+| 7  | iscsitarget | false  | STOPPED | <empty list> |
+| 9  | nfs         | false  | STOPPED | <empty list> |
+| 10 | snmp        | false  | STOPPED | <empty list> |
+| 11 | ssh         | false  | STOPPED | 98638        |
+| 14 | ups         | false  | STOPPED | <empty list> |
+| 18 | smartd      | false  | STOPPED | 3079         |
+| 26 | glusterd    | false  | STOPPED | <empty list> |
++----+-------------+--------+---------+--------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Reload Command
+
+The `reload` command reloads a specified service.
+
+*Possible states:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true` | Indicates the service reloaded. |
+| `false`| Indicates the service did not reload. |
+{{< /truetable >}}
+
+{{< expand "Reloading a Service" "v" >}}
+The `reload` command requires you to include the `service` option in the command string. After entering the command correctly, it returns either a true or false output.
+
+From the CLI prompt, enter:
+
+<code>service reload service=<i>name</i></code>
+
+From the **service** prompt, enter:
+
+<code>reload service=<i>name</i></code>
+
+Where:
+* `name` is the service name. For example, enter ssh for the ssh service.
+
+{{< expand "Command Example" "v" >}}
+```
+reload service=ssh
+true
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Restart Command
+
+The `restart` command restarts a specified service.
+
+*Possible returns:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true` | The service restarted successfully. |
+| `false`| The service did not restart. |
+{{< /truetable >}}
+
+{{< expand "Restarting a Service" "v" >}}
+The `restart` command requires you to include the `service` option in the command string. After entering the command correctly, it returns either a true or false output.
+
+From the CLI prompt, enter:
+
+<code>service restart service=<i>name</i></code>
+
+From the **service** prompt, enter:
+
+<code>restart service=<i>name</i></code>
+
+Where:
+* `name` is the service name. For example, enter ssh for the ssh service.
+
+{{< expand "Command Example" "v" >}}
+```
+restart service=ssh
+true
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Start Command
+
+The `start` command restarts a specified service.
+
+*Possible returns:*
```

Comment:
same comment as above and for any other command with this type of table. It is better placed with the details about the command in the expand.